src/Icon/FAIcon.stories.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { ThemeProvider } from 'styled-components';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import { expect } from '@storybook/jest';
+import { faRobot } from '@fortawesome/free-solid-svg-icons';
+import { theme } from '../shared/theme';
+import { FAIcon } from './FAIcon';
+import { FAIconProps } from './types';
+
+export default {
+  title: 'Components/Icon/FAIcon',
+  component: FAIcon,
+  argTypes: {
+    color: {
+      control: {
+        type: 'select',
+      },
+      defaultValue: 'blue.main',
```

Comment:
There is a warning regarding `defaultValue` being deprecated, in order to fix it the `defaultValue` would have to use the new API and set the default value like this:

```
 argTypes: {
    color: {
      control: {
        type: 'select',
      },
      ...
    },
},
args: {
  color: 'blue.main',
}
```
 
More info [here](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#no-longer-inferring-default-values-of-args)

Reply:
Nice, I will change it.